boot/freeldr/freeldr/ntldr/winldr.c

```diff
@@ -1251,6 +1251,29 @@ LoadAndBootWindowsCommon(
     /* "Stop all motors", change videomode */
     MachPrepareForReactOS();
 
+    /* Show the "debug mode" notice if needed */
```

Comment:
> `#if KDBG`

@HBelusca, Could we add `#if DBG` or would that be inappropriate in this very case?

Reply:
It should be possible to boot in debug mode even in release versions, since it's possible on Windows.
The check should be for KD support instead, because GCC builds don't support it.

Reply:
@xmine64 is correct.